File: tests/fixtures/lua/primitive_obsession.lua
function createRect(x, y, width, height, color, opacity)
    local rect = {}
    rect.x = x
    rect.y = y
    rect.width = width
    rect.height = height
    rect.color = color
    rect.opacity = opacity
    return rect
end

function simpleAdd(a, b)
    return a + b
end

File: tests/fixtures/objc/primitive_obsession.m
#import <Foundation/Foundation.h>

@implementation Renderer

- (void)draw:(NSInteger)x y:(NSInteger)y width:(NSInteger)w height:(NSInteger)h color:(NSInteger)c {
    NSLog(@"draw at %ld,%ld size %ldx%ld color %ld", (long)x, (long)y, (long)w, (long)h, (long)c);
}

@end
